JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.csv

```diff
@@ -45,7 +45,7 @@ hasTotalRoofArea,Object Property,,,http://www.purl.org/oema/infrastructure/Build
 hasPVsuitableRoofArea,Object Property,,,http://www.purl.org/oema/infrastructure/Building,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,A relationship between a building and its roof area suitable for photovoltaics installation.,https://www.theworldavatar.com/kg/ontobuiltenv/,has roof area suitable for PV
 hasPVsuitableRoofArea,Object Property,,,http://www.purl.org/oema/infrastructure/Building,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,exactly 1,,https://www.theworldavatar.com/kg/ontobuiltenv/,has roof area suitable for PV
 http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Area UNION http://www.ontology-of-units-of-measure.org/resource/om-2/Height UNION http://www.ontology-of-units-of-measure.org/resource/om-2/AmountOfMoney UNION AveragePricePerSqm,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has value
-http://www.ontology-of-units-of-measure.org/resource/om-2/hasNumericalValue,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,float,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has numerical value
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasNumericalValue,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure UNION http://www.ontology-of-units-of-measure.org/resource/om-2/Cost,float,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has numerical value
```

Comment:
Redefining someone else's classes/properties is not generally a good idea and should be avoided if at all possible. I would recommend removing such re-definitions. There seem to be several in this file.

Also, I believe `om:hasValue`, `om:hasNumericalValue`, and `om:hasUnit` can be used without having to change any definitions.

Reply:
Having discussed this with @yjytt59, we have concluded that, since this is an existing problem and not one that was introduced in this PR, we leave it as it is.

However, please do add a `README.md` file that explains this problem and notes this as an open TO-DO item.

Reply:
I already add a `README.md` to explain this issue and the current solution in sea level rise project.